serl_robot_infra/franka_env/envs/relative_env.py

```diff
@@ -44,7 +44,7 @@ def step(self, action: np.ndarray):
 
         # this is to convert the spacemouse intervention action
         if "intervene_action" in info:
-            info["intervene_action"] = self.transform_action(info["intervene_action"])
+            info["intervene_action"] = self.transform_action_inv(info["intervene_action"])
```

Comment:
same for other function in this class, please indicate in function docstring the leading six dimension are [\delta x, \delta y, ...] last one is gripper open/close (discrete?)